pricelist_by_category_qty/README.rst

```diff
@@ -0,0 +1,75 @@
+==========================================
+Discount by quantities of product category
+==========================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsale--workflow-lightgray.png?logo=github
+    :target: https://github.com/OCA/sale-workflow/tree/12.0-add-pricelist_by_category_qty/pricelist_by_category_qty
+    :alt: OCA/sale-workflow
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/sale-workflow-12-0-add-pricelist_by_category_qty/sale-workflow-12-0-add-pricelist_by_category_qty-pricelist_by_category_qty
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/167/12.0-add-pricelist_by_category_qty
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module will add a button inside the sale order and will compute the discount not by line but by quantities of the same product category, in the same sale order. e.g.: You've defined 10% discount for Category A for 100 PCE min in the List price. 
+
+You try to sell 50 Pieces of Product A and 50 Pieces of Product B (defined on Category A). 
+The OnChange in the line will compute the price of the product as usual. 
+Then you click on the new button. 
+The unit price will be updated as you've sold 50+50 = 100 pieces of the same category! => you will have 10% discount on both lines !
+
+This module was originally developed in v7.0 by the company Julius : https://github.com/julius-network-solutions/julius-openobject-addons/tree/master/pricelist_by_category_qty
```

Comment:
I would improve the README. let's say... Do the user need to activate pricelists and/or pricelist by formula, for example?

Reply:
Yes it's not perfect but as a first contribution, I think we should encourage new people releasing as it. It can be improved for future versions in many corners

Reply:
Sure it will be, no problem.

I'm not talking about an improvement but the minimal configuration steps of the README file. Do you know what I mean?